code/modules/research/designs/mechfab_designs.dm

```diff
@@ -913,51 +913,53 @@
 	name = "industrial suit control module assembly"
 	desc = "An assembly for a heavy, powerful rig used by construction crews and mining corporations."
 	id = "rig_industrial"
-	materials = list(DEFAULT_WALL_MATERIAL = 30000, "glass" = 12500)
+	materials = list(DEFAULT_WALL_MATERIAL = 25000, "glass" = 12500, "silver" = 3500, "gold" = 3500, "uranium" = 5550)
```

Comment:
Same goes for industrial RIG, it was intended to be printable without involvement of mining. Why are you even adding uranium here? Silver and gold are designed for combat and stealth rigs.